Showcase/ShowcaseTests/__Snapshots__/OUDSTokensBorderUITests/OUDSTokenBorderUITests.swift

```diff
@@ -38,14 +38,13 @@ final class OUDSTokensBorderUITests: XCTestCase {
     /// It iterates through all `NamedBorder` cases, rendering each illustration in a `UIHostingController`
     /// and captures a snapshot. The snapshot is saved with a name indicating the border, theme, and color scheme.
     @MainActor func testAllBordersOrangeThemeLight() {
```

Comment:
i think it should be Peter to factorize
crerate a :
- private testBorderWidth(for theme, in userInterfaceStyle)
- private testBorderRadius(for theme, in userInterfaceStyle)
- private testBorderStyle(for theme, in userInterfaceStyle)

and they create :

testAllBodersTokenForOrangeThemeLight() calling previous fonction with right parameters
testAllBodersTokenForOrangeThemeDark() calling previous fonction with right parameters
testAllBodersTokenForInverseThemeLight() calling previous fonction with right parameters
testAllBodersTokenForInverseThemeDark() calling previous fonction with right parameters

And try to apply the same logic for all tests

Reply:
Be carefull, the name of snashoot files will change (renaming)